ClientLibrary/Snippets/Microsoft.TeamServices.Samples.Client/Wiki/WikiPagesSample.cs

```diff
@@ -0,0 +1,182 @@
+
+            WikiPageResponse somePageResponse = wikiClient.GetPageAsync(
+                project: wiki.ProjectId,
+                wikiIdentifier: wiki.Name,
+                path: somePagePath).SyncResult();
+
+            Console.WriteLine("Retrieved page '{0}' metadata in wiki '{1}'", somePagePath, wiki.Name);
+
+            return somePageResponse;
+        }
+
+        [ClientSampleMethod]
+        public WikiPageResponse GetWikiPageMetadataWithContent()
+        {
+            VssConnection connection = this.Context.Connection;
+            WikiHttpClient wikiClient = connection.GetClient<WikiHttpClient>();
+
+            WikiV2 wiki = Helpers.FindOrCreateProjectWiki(this.Context);
+            string somePagePath = Helpers.GetAnyWikiPagePath(this.Context, wiki);
+
+            WikiPageResponse WikiPageResponse = wikiClient.GetPageAsync(
+                project: wiki.ProjectId,
+                wikiIdentifier: wiki.Id,
+                path: somePagePath,
+                includeContent: true).SyncResult();
+
+            Console.WriteLine("Retrieved page '{0}' metadata in wiki '{1}' with content '{2}'", WikiPageResponse.Page.Path, wiki.Name, WikiPageResponse.Page.Content);
+
+            return WikiPageResponse;
+        }
+
+        [ClientSampleMethod]
+        public WikiPageResponse GetWikiPageAndSubPages()
+        {
+            VssConnection connection = this.Context.Connection;
+            WikiHttpClient wikiClient = connection.GetClient<WikiHttpClient>();
+
+            WikiV2 wiki = Helpers.FindOrCreateProjectWiki(this.Context);
+            WikiPageResponse rootPageResponse = wikiClient.GetPageAsync(
+                project: wiki.ProjectId,
+                wikiIdentifier: wiki.Name,
+                path: "/",
+                recursionLevel: VersionControlRecursionType.OneLevel).SyncResult();
+
+            Console.WriteLine("Retrieved the following subpages for the root page:");
+            foreach (WikiPage subPage in rootPageResponse.Page.SubPages)
+            {
+                Console.WriteLine("Sub-page : '{0}'", subPage.Path);
+            }
+
+            return rootPageResponse;
+        }
+
+        [ClientSampleMethod]
+        public string GetWikiPageText()
+        {
+            VssConnection connection = this.Context.Connection;
+            WikiHttpClient wikiClient = connection.GetClient<WikiHttpClient>();
+
+            WikiV2 wiki = Helpers.FindOrCreateProjectWiki(this.Context);
+            string somePagePath = Helpers.GetAnyWikiPagePath(this.Context, wiki);
+
+            using (var reader = new StreamReader(wikiClient.GetPageTextAsync(
+                project: wiki.ProjectId,
+                wikiIdentifier: wiki.Id,
+                path: somePagePath).SyncResult()))
+            {
+                string pageContent = reader.ReadToEnd();
+                Console.WriteLine("Retrieved page '{0}' in wiki '{1}' with content '{2}'", somePagePath, wiki.Name, pageContent);
```

Comment:
switch all `Console.WriteLine` to `Context.Log`. Shouldn't have to change any of the parameters (and I believe Context.Log likely does a Console.WriteLine, but we do this so we can alter this type of output or do something else with it in teh future).